juspay/hyperswitch-control-center
# refactor: recon - update recon resources for rendering sidebar values based on permissions

## Type of Change




## Description
Described in https://github.com/juspay/hyperswitch/issues/6524

## Motivation and Context
Helps disable the recon related tabs which are not accessible to an user.

## How did you test it?
- Create an user (org_admin)
- Invite another user with granular permissions (recon reports view)
- Check the rendered UI

Org user (with all the permissions)
<img width="1728" alt="Screenshot 2024-11-25 at 1 04 02 PM" src="https://github.com/user-attachments/assets/db98f65d-b779-40f4-b943-09ed950dfbeb">

Invited user (with only view permissions)
<img width="1728" alt="Screenshot 2024-11-25 at 1 04 22 PM" src="https://github.com/user-attachments/assets/c4dafe5a-6821-47c6-9283-f286fef45323">

## Where to test it?

## Checklist
